diff --git a/ports/libgnutls/portfile.cmake b/ports/libgnutls/portfile.cmake
index 94302863e..be62f9e22 100644
--- a/ports/libgnutls/portfile.cmake
+++ b/ports/libgnutls/portfile.cmake
@@ -14,6 +14,7 @@ vcpkg_extract_source_archive(SOURCE_PATH
     SOURCE_BASE "v${VERSION}"
     PATCHES
         use-gmp-pkgconfig.patch
+        clang-gettime.patch
 )
 
 vcpkg_list(SET options)
@@ -33,6 +34,8 @@ if(VCPKG_TARGET_IS_OSX)
 endif()
 
 set(ENV{GTKDOCIZE} true) # true, the program
+set(ENV{CFLAGS} " -D_POSIX_C_SOURCE -Wno-int-conversion")
+set(ENV{CXXFLAGS} " -D_POSIX_C_SOURCE -Wno-int-conversion")
 vcpkg_configure_make(
     SOURCE_PATH "${SOURCE_PATH}"
     AUTOCONFIG
@@ -50,6 +53,7 @@ vcpkg_configure_make(
         --with-tpm=no
         --with-tpm2=no
         --with-zstd=no
+        --disable-hardware-acceleration
         ${options}
         YACC=false # false, the program - not used here
     OPTIONS_DEBUG
